projects/lights_control/inc/lights_events.h

```diff
@@ -28,3 +36,11 @@ typedef enum {
   LIGHTS_EVENT_GPIO_PERIPHERAL_SYNC,
   NUM_LIGHTS_EVENT_GPIO_PERIPHERALS
 } LightsEventGpioPeripheral;
+
+// Possible data fields to be used with a LIGHTS_EVENT_SIGNAL_* event.
+typedef enum {
+  LIGHTS_EVENT_SIGNAL_MODE_LEFT = 0,
+  LIGHTS_EVENT_SIGNAL_MODE_RIGHT,
+  LIGHTS_EVENT_SIGNAL_MODE_HAZARD,
+  NUM_LIGHTS_EVENT_SIGNAL_MODES
+} LightsEventSignalMode;
```

Comment:
Technically your file is named `lights_events` but you're using `LightsEvent`

🤔

Reply:
🤔

Reply:
I'll change this with the next PR that's gonna be merged.